env/lib/python3.12/site-packages/pip/__pip-runner__.py

```
@@ -0,0 +1,50 @@
+
+if sys.version_info[:2] < PYTHON_REQUIRES:
+    raise SystemExit(
+        "This version of pip does not support python {} (requires >={}).".format(
+            version_str(sys.version_info[:2]), version_str(PYTHON_REQUIRES)
+        )
+    )
+
+# From here on, we can use Python 3 features, but the syntax must remain
+# Python 2 compatible.
+
+import runpy  # noqa: E402
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)